scripts/scil_evaluate_connectivity_measures.py

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Evaluate graph theory measures from connectivity matrices.
+A length weighted and a streamline count weighted matrix are required since
+some measures require one or the other.
+
+This script evaluate the measure one subject at the time, to generate a
+population dictionary (similarly to the other scil_evaluate_*.py ) use the
+--append_json option as well as using the same output filename.
+
+Some measures output one value per node, the default behavior is to average
+them all into a single value. To obtain all values as a list use the
+--node_wise_as_list option.
+
+The computed connectivity measures are:
+centrality, modularity, assortativity, participation, clustering, degree
+nodal_strength, local_efficiency, global_efficiency, density, rich_club
+path_length, edge_count
+
+For more details about the measures, please refer to
+- https://sites.google.com/site/bctnet/measures
+- https://github.com/aestrivex/bctpy/wiki
+"""
+
+import argparse
+import json
+import logging
+import os
+import warnings
+
+import bct
+import numpy as np
+
+from scilpy.io.utils import (add_json_args,
+                             add_overwrite_arg,
+                             add_verbose_arg,
+                             assert_inputs_exist,
+                             assert_outputs_exist,
+                             load_matrix_in_any_format,
+                             save_matrix_in_any_format)
+
+
+EPILOG = """
+References:
+    [1] Rubinov, Mikail, and Olaf Sporns. "Complex network measures of brain
+        connectivity: uses and interpretations." Neuroimage 52.3 (2010):
+        1059-1069.
+"""
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(description=__doc__,
+                                formatter_class=argparse.RawTextHelpFormatter,
+                                epilog=EPILOG)
+    p.add_argument('in_length_matrix',
+                   help='Input length weighted matrix.')
```

Comment:
add `(npy).` ? I know once you get there it should be obvious but I think it's still useful. Tell me what do you think.